test/e2e/info_test.go

```diff
@@ -49,4 +53,32 @@ var _ = Describe("Podman Info", func() {
 		Expect(session.ExitCode()).To(Equal(0))
 		Expect(session.IsJSONOutputValid()).To(BeTrue())
 	})
+	It("podman info rootless storage path", func() {
+		if !rootless.IsRootless() {
+			Skip("test rootless_storage_path for rootless")
+		}
+		SkipIfRemote()
+		configPath := "/etc/containers/storage.conf"
+		desPath := "/etc/containers/storage.conf.test"
+		cmd := exec.Command("sudo", "cp", configPath, desPath)
+		out, err := cmd.CombinedOutput()
+		fmt.Println(string(out))
+		Expect(err).To(BeNil())
+
+		rootStoragePath := `"/tmp/$HOME/$USER/$UID"`
+		cmd = exec.Command("sudo", "sed", "-i", "-e", "/rootless_storage_path/d", "-e", fmt.Sprintf("/\\[storage\\]/a rootless_storage_path = %s", rootStoragePath), configPath)
+		err = cmd.Run()
+		Expect(err).To(BeNil())
+
+		expect := filepath.Join("/tmp", os.Getenv("HOME"), os.Getenv("USER"), os.Getenv("UID"))
+		podmanPath := podmanTest.PodmanTest.PodmanBinary
+		cmd = exec.Command(podmanPath, "info", "--format", "{{.store.GraphRoot}}")
+		out, err = cmd.Output()
+		Expect(err).To(BeNil())
+		Expect(string(out)).To(ContainSubstring(expect))
+
+		cmd = exec.Command("sudo", "cp", desPath, configPath)
```

Comment:
I think you can and should do this test without `sudo` and especially without modifying system files. But for future reference: should you ever really need to modify a system file, this is not enough. You need to write a `restore` that is safe, robust, bulletproof - that will run even if the `Expect`s fail, if the test has an error. I don't know the Go way to do this, maybe `defer`?